src/components/map-controllers/SingleMapController/SingleMapController.js

```diff
@@ -231,6 +231,7 @@ export default class SingleMapController extends React.Component {
         </Panel.Title>
         </Panel.Heading>
         <Panel.Body className={styles.mapcontroller}>
+          {/*Foo*/}
```

Comment:
I'm sure there's some sort of style black magic to having this here, but a comment explaining what would be nice. :)

Reply:
Absolutely not. Cruft of the worst and most hackneyed order. Gone!